bitnami/thanos/templates/ruler/statefulset.yaml

```
@@ -153,7 +153,7 @@ spec:
             - --label=ruler_cluster="{{ .Values.ruler.clusterName }}"
             - --alert.label-drop={{ .Values.ruler.replicaLabel }}
             - --objstore.config-file=/conf/objstore/objstore.yml
-            - --rule-file="{{ .Values.ruler.ruleFile }}"
+            - --rule-file={{ .Values.ruler.ruleFile }}
```

Have you tried this?

```suggestion
            - --rule-file={{ .Values.ruler.ruleFile | quote}}
```

---

the quotations have to be removed completely

---

Just for the record, I changed locally adding your mod @fmulero and the end result would still add the undesired quotation marks, breaking the required globing, the quotes must be removed all along.

```
$ helm template thanos bitnami/thanos/ --set ruler.rule-file='/conf/rules/*.yml' --set ruler.enabled=true --set ruler.existingConfigmap=myconfigmap --set existingObjstoreSecret=madeup --set ruler.alertmanagers={'localhost:9093'} | grep -C 5 "rule-file"
            - --alert.query-url=http://thanos-query.civitatis.svc.cluster.local:9090
            - --label=replica="$(POD_NAME)"
            - --label=ruler_cluster=""
            - --alert.label-drop=replica
            - --objstore.config-file=/conf/objstore/objstore.yml
            - --rule-file="/conf/rules/*.yml"
          env:
            - name: POD_NAME
              valueFrom:
                fieldRef:
                  fieldPath: metadata.name
```